src/cmd/cmd.cpp

```diff
@@ -339,7 +339,14 @@ int main(int argc, char **argv)
         qFatal("Could not initialize account!");
         return EXIT_FAILURE;
     }
-    // check if the webDAV path was added to the url and append if not.
+
+    if (options.target_url.contains("webdav")) {
+        qWarning("Webdav is detected, even though it is deprecated.");
+        std::cout << "Warning: Webdav is deprecated and is going to be removed soon. Please, use dav path for a host. Example:" << std::endl
+                  << "https://yourusername:youruserpassword@yourhost/remote.php/dav/files/remotefolderpath" << std::endl;
+    }
+
+    // check if the dav path was added to the url and append if not.
```

Comment:
What is the relation between this new comment and the code just after it ?
I am missing the point here

Reply:
@mgallien No relation at all. It was there before and I don't need to remove it for my PR. It may seem to you that it was somehow modified, but, in fact, it is the wrong impression of Git diff. My comment was added before this line. So, I am now confused about, what concern you have and what I need to change?

Reply:
I will have a look again